gatsby-node.js

```diff
@@ -137,15 +138,18 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
 
   allMarkdownRemark.edges.forEach(({ node }) => {
     const {
+      fileAbsolutePath,
       frontmatter: { template },
       fields: { slug },
     } = node;
+    const fileRelativePath = fileAbsolutePath.match(/src\/.*/g)[0];
```

Comment:
We've been using [this function](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/gatsby-node.js#L131-L132) to get the relative path from the absolute path. Should we be consistent?

Reply:
oh right. i saw that earlier, but spaced it when i got it working. but yeah, makes sense to me to use and be consistent